client/network/src/service.rs

```diff
@@ -1565,6 +1567,7 @@ impl<B: BlockT + 'static, H: ExHashT> Future for NetworkWorker<B, H> {
 					if let Some(s) = peers_notifications_sinks.get_mut(&(remote, protocol)) {
 						*s = notifications_sink;
 					} else {
+						debug_assert!(false);
```

Comment:
I would put it after log::error, because I would like to log the error and then fail.